opencti-platform/opencti-graphql/src/database/redis.ts

```diff
@@ -441,6 +441,12 @@ const buildMergeEvent = async (user: AuthUser, previous: StoreObject, instance:
   const message = generateMergeMessage(instance, sourceEntities);
   const previousStix = convertStoreToStix(previous) as StixCoreObject;
   const currentStix = convertStoreToStix(instance) as StixCoreObject;
+  if ('extensions' in currentStix
+      && 'extension-definition--ea279b3e-5c71-4632-ac08-831c66a786ba' in currentStix.extensions
+      && 'stix_ids' in currentStix.extensions['extension-definition--ea279b3e-5c71-4632-ac08-831c66a786ba']
+      && currentStix.extensions['extension-definition--ea279b3e-5c71-4632-ac08-831c66a786ba'].stix_ids.length > 1000) {
+    currentStix.extensions['extension-definition--ea279b3e-5c71-4632-ac08-831c66a786ba'].stix_ids = currentStix.extensions['extension-definition--ea279b3e-5c71-4632-ac08-831c66a786ba'].stix_ids.slice(-1000, 0);
```

Comment:
.slice(-1000, 0) returns an empty array, shouldn't it be .slice(-1000) if we want the last 1000 elements of the array ?